### PR TITLE
Visi aha defnet epiphyte

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: synapse/lib/aha.py [hunk 1]
```diff
@@ -1221,8 +1230,8 @@ async def signHostCsr(self, csrtext, signas=None, sans=None):
 
         hostname = xcsr.subject.get_attributes_for_oid(c_x509.NameOID.COMMON_NAME)[0].value
 
-        hostpath = s_common.genpath(self.dirn, 'certs', 'hosts', f'{hostname}.crt')
-        if os.path.isfile(hostpath):
+        hostpath = self.certdir.getHostCertPath(hostname)
+        if hostpath:
```
Comment (invisig0th) [inline synapse/lib/aha.py:1234]: Should these be `is not None` checks?

File: synapse/lib/aha.py [hunk 2]
```diff
@@ -1240,8 +1249,8 @@ async def signUserCsr(self, csrtext, signas=None):
 
         username = xcsr.subject.get_attributes_for_oid(c_x509.NameOID.COMMON_NAME)[0].value
 
-        userpath = s_common.genpath(self.dirn, 'certs', 'users', f'{username}.crt')
-        if os.path.isfile(userpath):
+        userpath = self.certdir.getUserCertPath(username)
+        if userpath:
```
Comment (invisig0th) [inline synapse/lib/aha.py:1253]: here too

File: synapse/lib/aha.py [hunk 3]
```diff
@@ -1278,8 +1287,7 @@ def getMyUrl(self, user='root'):
         return f'ssl://{host}:{port}?certname={user}@{network}'
 
     async def getAhaClone(self, iden):
-        lkey = s_common.uhex(iden)
-        byts = self.slab.get(lkey, db='aha:clones')
+        byts = self.slab.get(iden.encode(), db='aha:clones')
```
Comment (invisig0th) [inline synapse/lib/aha.py:1290]: I'd like to stick with the `uhex()` value ( and eventually convert the other uses to it as well )

Comment (vEpiphyte) [reply synapse/lib/aha.py:1290]: yup :) haven't changed them back yet

File: synapse/lib/aha.py (hunk 1 above)
Comment (vEpiphyte) [reply synapse/lib/aha.py]: We could go either way; we're either getting a `None` or a `str` with some text in it. changed them to be inline with some of the other certdir checks in aha.py.

**Pull request merged**